packages/mc-http-server/bin/start-mc.js

```diff
@@ -1,52 +1,76 @@
 #!/usr/bin/env node
 
 /* eslint-disable no-console,global-require */
+
 const fs = require('fs');
 const path = require('path');
-const shell = require('shelljs');
-const fetch = require('node-fetch');
-const {
-  replaceHtmlPlaceholders,
-} = require('@commercetools-frontend/mc-html-template');
+const mri = require('mri');
+const shelljs = require('shelljs');
+const { compileHtml } = require('@commercetools-frontend/mc-html-template');
 const {
-  packageLocation: assetsPath,
+  packageLocation: applicationStaticAssetsPath,
 } = require('@commercetools-frontend/assets');
 const startServer = require('../server');
-const options = require('../load-options');
+
+const flags = mri(process.argv.slice(2), { alias: { help: ['h'] } });
```

Comment:
The CLI args parsing was previously defined in `load-options` and has been now moved here